openid-4-verifiable-presentations-1_0.md

```diff
@@ -320,6 +320,24 @@ The following is a non-normative example of a transaction data content, after ba
 }
 ```
 
+`attachments`:
+: OPTIONAL. Array of objects, where each object includes an extra resource that needs to be passed to the Wallet. These resources are intended to provide additional context to the Wallet or the End-User. The Wallet MUST ignore any unrecognized attachment formats. Each object contains the following parameters:
+
+    * `format`: REQUIRED: String that identifies the format of the attachment. The specific values are out of scope of this specification. It is RECOMMENDED to use collision-resistant names for type values.
+    * `data`: REQUIRED: The structure and content of data are format-dependent and not constrained by this specification.
+    * `credential_ids`: OPTIONAL. Array of strings each referencing a Credential requested by the Verifier for which the attachment is relevant. Each string matches the `id` field in a DCQL Credential Query. If omitted the attachment is relevant to all requested credentials.
+
+See (#attachments-in-authorization-requests) for more details.
+
+The following is a non-normative example of an attachment object:
+
+```
+{
+  "format": "eudi_registration_certificate",
```

Comment:
That suprises me. I would have expected this to be "jwt" (for example) and then an additional attribute "type" determining the type as "eudi_registration_certificate". The type would also be determined from `typ`JSON header or a `vct` claim within the JWT.

Reply:
I don't see the case why why an eco system should define different formats (json, string) for the same type. When defining a specific attachment, the unique identifier like `eudi_registration_certificate` will be defined in the ecosystem how to process it (so also how to decode the information if required and how to consume the information).

Defining a generic `format` field with the values like `string` | `json`, ... does not seam good, especially since the type needs to be string based to embed it in json. That's why the iana media type was removed for the format field and replaced by a unique identifier.

Reply:
used type, format value is not relevant